### DISCUSSION TITLE
SLBC TV [SL]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#451] [Show and tell]
### Channel ID

SLBCTV.sl

### Stream URL (optional)

https://ythls.onrender.com/channel/UCEtHh4-TrhA-4BYgoGyCONQ.m3u8

### Notes (optional)

720p / Not 24/7

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Sphinxroot** [comment]
https://www.youtube.com/@slbcnews5245/streams  not mutch  live :/
